llvm/include/llvm/BinaryFormat/ELF.h

```diff
@@ -699,11 +699,22 @@ enum {
 
 // SPARC Specific e_flags
 enum : unsigned {
+  // ELF extension mask.
+  // All values are available for EM_SPARC32PLUS & EM_SPARCV9 objects,
```

Comment:
It looks like you've hand-wrapped this comment, and so it is wrapped prematurely in a number of places. Please let clang-format do the wrapping, or consistently wrap at 80 characters. NB: clang-format won't "fix" incorrect wrapping, so the easiest thing to do is just delete all the new lines and run clang-format on the comment.